src/app/components/container/container.tsx

```diff
@@ -0,0 +1,11 @@
+export async function NHSNotifyContainer({
+  children,
+}: {
+  children: React.ReactNode;
+}) {
+  return (
+    <div className='nhsuk-width-container app-width-container'>
+      <div className='nhsuk-main-wrapper'>{children}</div>
```

Comment:
This is the suggested main wrapper from the nhs - https://service-manual.nhs.uk/design-system/styles/layout#main-content